VAMobile/documentation/docs/Engineering/DevOps/Creating a Firebase Project.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Creating a Firebase Project
+postion: 3
+---
+
+## Introduction
+
+This document will go through the steps of creating a firebase project.
+
+:::info
+You will need access to the `va-mobile-app` in Firebase before you can complete this task. Please reach out to your manager if you do not have access.
+:::
+
+Firebase is used for analytics as well as remote config options. With these options we are able to set feature flags for options within the app. We can also set this to only push to certain versions as well as a certain percentage of the userbase at one time.
+
+## Create Project
+
+**1.** After you are granted access to the Firebase project, click on the project at the top of your screen and then click `Create a project`.
+
+![Create Firebase Project](/img/engineering/devops/firebase/fbCreateProject.png)
+
+**2.** Name your project and click `Continue`.
+
+![Name Firebase Project](/img/engineering/devops/firebase/fbNameProject.png)
+
+**3.** On the AI assistance screen, click `Continue`.
+
+![Gemini Enable](/img/engineering/devops/firebase/fbGeminiEnable.png)
+
+**4.** Since this is not Production, we want to disable analytics to start with. We can enable this later if needed. click the Check mark next to `Enable Google Analytics for this project` and click `Create Project`.
```

Comment:
"Since this is not Production" indicates that we're just documenting what we did to set up staging